src/afl-fuzz-run.c

```diff
@@ -921,6 +922,53 @@ common_fuzz_stuff(afl_state_t *afl, u8 *out_buf, u32 len) {
 
   fault = fuzz_run_target(afl, &afl->fsrv, afl->fsrv.exec_tmout);
 
+  int write_event_log = 1;
+  if (write_event_log) {
+    struct fuzz_event_buffer *buffer = &afl->event_buffer;
+    u64                       buf_size = buffer->size;
+    u64                       buf_max_len = 16384;
+    if (buf_max_len <= buf_size) {
+      // TODO(wuestholz): Trigger this before AFL terminates.
+      FILE *f = buffer->log_file;
+      if (!f) {
+        f = fopen(alloc_printf("%s/fuzz_event_log.csv", afl->out_dir), "w");
+        if (!f) { PFATAL("Unable to create 'fuzz_event_log.csv'"); }
+        buffer->log_file = f;
+      }
+
+      for (u64 i = 0; i < buf_max_len; i++) {
```

Comment:
Did you mean
```
for (u64 i = 0; i < buf_size; i++) {
```
Do you need i to be u64?

Reply:
I think `buf_size` should be equal to `buf_max_len` here, but I can use `buf_size` instead.

The reason for using u64 was mostly that I had picked u64 for `buffer->size`. However, u32 should do.